package haditya.kotlin.restful.error

class UnauthorizedException:Exception() {
}